pandas/tests/arrays/boolean/test_misc.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
```

Comment:
can we avoid "test_misc"?  assuming this refactor is worthwhile, might as well have test_unary.py, test_repr.py, ...